include/grpc/grpc.h

```diff
@@ -351,6 +351,23 @@ typedef struct grpc_op {
   } data;
 } grpc_op;
 
+/* Propagation bits: this can be bitwise or-ed to form propagation_mask for
+ * grpc_call */
+/** Propagate deadline */
+#define GRPC_PROPAGATE_DEADLINE ((gpr_uint32)1)
+/** Propagate census context */
+#define GRPC_PROPAGATE_CENSUS_CONTEXT ((gpr_uint32)2)
```

Comment:
You asked: What if I want to make one RPC without tracing propagation (but perhaps with deadline propagation?)

Good question. This would seem to be the best way of doing this (via updating the trace_mask when we start the operation). It raises another issue though, in that we use the census context for more than just tracing - what if I want to disable tracing, but keep stats collection enabled for this request? There are probably a couple of options here:
1. Use the GRPC_PROPAGATE_CENSUS_CONTEXT flag to {en/dis}able all features of census that require propagation.
2. Explicitly state that the GRPC_PROPAGATE_CENSUS_CONTEXT flag can only be used to disable tracing (leaving other census features that require propagation untouched).
3. Add GRPC_PROPAGATE_CENSUS_CONTEXT_X flags for the other census features (RPC stats + CPU accounting) that are done per-context. 

3 would be my preference.